### PR TITLE
[zh]update health-checks.md.

### PR DESCRIPTION
**Pull request by yuandongx**
Just formate changed.

/assign @howieyuen
/assign @tengqm

### CODE REVIEW

**Comment by howieyuen** [comment]
so indents can also be translated to code blocks?
@tengqm

**Comment by tengqm** [comment]
This is an invalid fix.
The correct fix is to change the `shell` to `none` or just remove it.

@howieyuen Yes. Indentation by 4 spaces will be treated as implicit code blocks, which is a practice we discourage.

**Comment by yuandongx** [comment]
@tengqm  @howieyuen 
I just keep the format of Chinese and English consistent.
I'll close the process if you think it's not ok.
image

**Comment by tengqm** [comment]
@yuandongx Pls do as I suggested, remove 'shell' from the code fence. That is all we need.

**Comment by xichengliudui** [comment]
> @yuandongx Pls do as I suggested, remove 'shell' from the code fence. That is all we need.

**Comment by xichengliudui** [comment]
> @yuandongx Pls do as I suggested, remove 'shell' from the code fence. That is all we need.

+1

File: content/zh/docs/reference/using-api/health-checks.md [hunk 1]
```diff
@@ -51,41 +51,35 @@ This can be useful for a human operator to debug the current status of the Api s
 对于所有端点，都可以使用 `verbose` 参数来打印检查项以及检查状态。
 这对于操作人员调试 API 服务器的当前状态很有用，这些不打算给机器使用：
 
-```shell
-curl -k https://localhost:6443/livez?verbose
-```
+    curl -k https://localhost:6443/livez?verbose
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/health-checks.md:54]: don't do this!
We prefer explicit code fences to indentation based code blocks.

File: content/zh/docs/reference/using-api/health-checks.md [hunk 2]
```diff
@@ -65,27 +65,25 @@ kubectl get --raw='/readyz?verbose'
 <!-- The output will look like this: -->
 输出将如下所示：
 
-```shell
```
Comment (tengqm) [inline content/zh/docs/reference/using-api/health-checks.md:68]: just remove 'shell' would be okay, pls keep the code fence (three backtiqs) and the indentation.

Comment (yuandongx) [reply content/zh/docs/reference/using-api/health-checks.md:68]: 我看到了英文是使4个空格的缩进，所以也让中文用4个空格替换原来的`shell`。
这里 ` keep the code fence (three backtiqs) and the indentation` 就让人有点困惑了.....

Comment (tengqm) [reply content/zh/docs/reference/using-api/health-checks.md:68]: Did you read my previous comment???

>  Yes. Indentation by 4 spaces will be treated as implicit code blocks, which is a practice we discourage.

Comment (yuandongx) [reply content/zh/docs/reference/using-api/health-checks.md:68]: @tengqm 
I saw that.
Sorry, there's something wrong with my understanding.

**Pull request merged**